components/tabs/src/index.js

```diff
@@ -0,0 +1,164 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+import { TEXT_COLOUR } from 'govuk-colours';
+import { spacing, typography } from '@govuk-react/lib';
+
+import List from './atoms/list';
+import Panel from './atoms/panel';
+import Tab from './atoms/tab';
+import Title from './atoms/title';
+
+const Tabs = styled('div')(
+  typography.font({ size: 19 }),
+  {
+    color: TEXT_COLOUR,
+    marginTop: spacing.responsive({ size: 1, property: 'margin', direction: 'top' }),
+    marginBottom: spacing.responsive({ size: 6, property: 'margin', direction: 'bottom' }),
+  },
+  spacing.withWhiteSpace(),
+);
+
+/**
+ *
+ * ### Import
+ * ```js
+ * import Tabs, { Tab, List, Panel, Title } from '@govuk-react/tabs';
```

Comment:
does this style of import actually work, given how `Tab`, `List`, `Panel` and `Title` are being exported

Also as we have a `Panel` component already it's arguably advisable to just have the example use `Tab.Panel` etc

Reply:
This style of import would not have worked, given the code at the time, as the sub-components were not named exports.

The sub-components are now named exports, but this part of the docs has been removed.